index.html

```diff
@@ -310,7 +310,29 @@ <h3>
                   if you'd like to </br>
                   chat please get in touch.
                 </h3>
-                <!-- place form -->
+                <!-- place form here -->
+                <section class="form-left-decoration"></section>
+                <section class="form-right-decoration"></section>
+                <section class="circle"></section>
+                <section class="form-inner">
+                  <form action="https://formspree.io/f/xoqzdylg" method="POST" class="decor">
+                    <ul>
+                      <li>
+                        <input type="text" maxlength="30" placeholder="Full name" name="userName" required/>
+                      </li>
+                      <li>
+                        <input type="email" placeholder="Email address" name="email" required/>
+                      </li>
+                      <li>
+                        <textarea maxlength="500" placeholder="Enter text here" rows="5" name="text" required></textarea>
+                      </li>
+                      <li>
+                        <br>
+                        <button type="submit">Get in touch</button>
+                      </li>
+                    </ul>
```

Comment:
Hello team, good job so far 😃. As a can see your from is working, but you need to fix some parts.

- [ ] Kindly, It could be better if you remove the `ul`, `li`, and `<br>` tags in your `form` , then you can use the `Flexbox` in order to lay out the elements on the page as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox.) 👍. You my use this form : 
````
<form>
      <input />
      <input />
      <textarea></textarea>
</form>
````

Reply:
Done...have updated the form and used flexbox as well

Reply:
Thank you